### PR TITLE
Feature/map screen

### LINKED ISSUE #2: Map
As a user, when I visit the home page I should see a map, descriptions and options, so that I can click on an option and start an encounter
- [x] [Map Image](#5)
- [x] [Clickable actions](#6)
- [x] [Display description](#7)
- [ ] [Triggers a change in encounter](#8)

### LINKED ISSUE #5: Map Image
The map screen should show an image of the overworld map

### LINKED ISSUE #6: Clickable actions/places
The map screen should have some different places to go available in the actions

### LINKED ISSUE #8: Display description
The map screen should show a description of the world including time of day, the atmosphere of your previous location, and any other relevant factors such as a descriptor of your health if it is low.

### PR DESCRIPTION
**Pull request by rubyr**
## Description

This branch aims to add the map view as well as the associated chores like a description and place names. It also adds react-router into the program for easy routing throughout the application.

Fixes #2 fixes #5 fixes #6 fixes #8 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

Branch still needs additional testing.
- [x] Description.test.js
- [x] Map.test.js
- [x] Actions.test.js
- [ ] App.test.js

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
    - testing tbd
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/Description/Description.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  // array of arrays, first element of each subarray is a query into the player's stats,
+  // second element is a bit of info to add to the player's description
+  // ex: details[0] takes in the player's hp and maxhp and if the hp is less then half
+  // of the maxhp, gives back a "you are hurt" message.
+  const details = [
+    [({ hp, maxhp }) => hp < maxhp / 2, "You are hurt. "],
+    [({ attack }) => attack < 3, "You feel weak. "],
+    [({ defense }) => defense < 3, "You feel defenseless. "],
+    [({ name }) => !name, "You feel a loss of identity. "],
+  ];
```
Comment (Corbinj22) [inline src/Description/Description.js:19]: You can tell you used to make games, these are literally the little things I wouldn't have thought about. 
Love the "You feel a loss of identity" 👍

File: src/Map/Map.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    ];
+
+    let formattedMap = mapImage;
+
+    for (const [re, sub] of colTags) {
+      formattedMap = formattedMap.replace(re, sub);
+    }
+
+    return formattedMap;
+  }, []);
```
Comment (Corbinj22) [inline src/Map/Map.js:22]: This is amazing! I was wondering how you wanted to accomplish this.

File: src/Description/Description.js (hunk 1 above)
Comment (rubyr) [reply src/Description/Description.js:19]: Yeah, I'd imagine we'd add to this as time goes on with more tidbits of information

File: src/Actions/Actions.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Actions.css";
+
+const Actions = () => {
+  return (
+    <div className="Actions">
+      <Link to="/places/forest">Go to the forest</Link>
```
Comment (Maxwell-Baird) [inline src/Actions/Actions.js:8]: So this how we are going to move across the map. Got it

**Pull request merged**